Gopkg.toml

```diff
@@ -46,12 +46,12 @@ required = [
   branch = "master"
   name = "github.com/giantswarm/backoff"
 
-[[constraint]]
-  branch = "master"
+[[override]]
```

Comment:
This should be `constraint` since it's a direct dependency. If project has e2esetup as transitive dependency too and it's not being resolved to same branch/version only then it should be added as override too.